test/e2e-test-application/e2e/test2/1-angular/external-micro-frontend.spec.js

```diff
@@ -0,0 +1,16 @@
+Cypress.env('RETRIES', 1);
+describe('Navigation', () => {
+  beforeEach(() => {
+    cy.visitLoggedIn('/');
+  });
+  describe('Normal navigating', () => {
+    it('Check if external micro frontend is available and an alert will be shown.', () => {
+      cy.visit('/projects/pr1/externalmf');
+      cy.wait(1000);
+      cy.get('[data-testid=luigi-alert]').should(
+        'have.class',
+        'fd-message-strip--success'
```

Comment:
The test seemed to fail for me, probably checking for an old type of fundamentals class name, I replaced this with `fd-alert--success` and it found it

Reply:
Tests were green for me. Did you run `lerna build` before running the tests?

Reply:
I had actually. Seems like I had forgotten to `symlink` 😅 . Works fine for me as well 🟢